components/math/longestStreak/Winner.tsx

```diff
@@ -1,66 +1,58 @@
-import { useMutation } from "@apollo/client";
+
 import React, { useState } from "react";
-import { RESET_GAME_LEVEL } from "../../../graphql/longestStreak/resetGameLevel";
+import Firework from "./Firework";
+
 
 export interface WinnerProps {
   text: string;
   onClick: () => void;
   onRestartClick: () => void;
   onSameLevelClick: () => void;
-  winner: string;
-  image: string;
-  user: any;
+
+  endOfGame: any;
```

Comment:
Shouldn't use the any type. What type is this?

Reply:
Changed to WinnerData (an array of strings)